docs/snapcraft-syntax.md

```diff
@@ -20,24 +20,33 @@ contain.
  * `config` (string)  
    Path to the runnable in snap that will be used as the [Snappy config
    interface](https://developer.ubuntu.com/snappy/guides/config-command/).
- * `services` (yaml subsection)  
-   A set of keys representing service names with values as defined by the
-   [Snappy packaging spec](https://developer.ubuntu.com/snappy/guides/packaging-format-apps/).
- * `binaries` (yaml subsection)  
-   A set of keys representing binary names with values as defined by the
-   [Snappy packaging spec](https://developer.ubuntu.com/snappy/guides/packaging-format-apps/).
+ * `apps` (yaml subsection)  
+   A map of keys for integration points for the system. Either as a daemon
+   or command line accessible binary.
+   * `command` (string)
+     Specifies the internal command to expose.
+   * `daemon` (string)
+     If present, integrates the runnable as a system service. Valid values are
+     forking and simple.
+   * `stop-command` (string)
+     Requires `daemon` to be specified and represents the command to run to
+     effectively stop the service.
+   * `stop-timeout` (integer)
+     Requires `daemon` to be specified. It is number in seconds to indicate how
+     much time to wait for the service to stop
  * `icon` (string)  
    Path to the icon that will be used for the snap.
- * `license` (yaml subsection)  
-   License the snap will carry.
-   * `text` (string)  
-     The license text for the snap itself.
-   * `accept-required` (boolean)  
-     If true, license acceptance is required for the package to be activated.
-     A good example for this one is the Sun JRE/JDK being bundled in a snap.
- * `framework-policy` (string)  
-   A relative path to a directory containing additional policies, used if
-   creating a framework and want to extend permissions to snap apps.
+ * `license` (string)  
+   Path to a license file.
+ * `license-agreement` (string)  
+   Requires `license` to be set. The only valid value for this entry is
+   `explicit` stating that if a license is present, acceptance to it must
```

Comment:
What about: stating that if a license is present, the user must accept it in order to install the snap.